### ISSUE TITLE
Clicking links in main window fails to do anything

### ISSUE THREAD

**Issue by alexmurray** [canonical/desktop-security-center#55]
The main window of the security center has the links 'Learn more' 'Give feedback' and 'Report issues' but clicking on any of them fails to do anything.

I am using revision 6 from the current stable channel on Ubuntu 24.10:

```
name:      desktop-security-center
summary:   Desktop Security Center
publisher: Canonical✓
store-url: https://snapcraft.io/desktop-security-center
license:   GPL-3.0
description: |
  Security Center UI for the desktop
commands:
  - desktop-security-center.security-center
snap-id:      FppXWunWzuRT2NUT9CwoBPNJNZBYOCk0
tracking:     latest/stable
refresh-date: today at 11:39 ACST
channels:
  latest/stable:    0+git.6896825 2024-08-28  (6) 11MB -
  latest/candidate: ↑                                  
  latest/beta:      ↑                                  
  latest/edge:      0+git.b9a3436 2024-09-04 (10) 11MB -
  1/stable:         0+git.6896825 2024-08-28  (6) 11MB -
  1/candidate:      ↑                                  
  1/beta:           ↑                                  
  1/edge:           0+git.b9a3436 2024-09-04 (10) 11MB -
installed:          0+git.6896825             (6) 11MB -
```

**Comment by sminez** [comment]
I think these are placeholders for now while we wait for UX to set up the pages that those links will be pointing at (covered in [this Jira issue](https://warthogs.atlassian.net/browse/UDENG-3896)). My understanding that the only thing outstanding at this point is the announcement post for "learn more" which should be coming soon.

**Comment by d-loose** [comment]
Fixed by #59
